docroot/modules/custom/sitenow_events/sitenow_events.module

```diff
@@ -214,18 +225,50 @@ function sitenow_events_preprocess_block(&$variables) {
           }
         }
 
-        foreach ($data['events'] as $value) {
+        // Base styles.
+        $styles = [
```

Comment:
I don't think we are using these classes anymore.

Reply:
removed